Makefile

```diff
@@ -1,10 +1,65 @@
-test:
+# COLORS
+RED=\033[0;31m
+GREEN=\033[0;32m
+ORANGE=\033[0;33m
+BLUE=\033[0;34m
+PURPLE=\033[0;35m
+CYAN=\033[0;36m
+LIGHTGRAY=\033[0;37m
+DARKGRAY=\033[1;30m
+YELLOW=\033[0;33m
+NC=\033[0m # No Color
+
+chip?=F4
+BSPConfig?=off
+
+# Used to select the MCU (default is STM32F413)
+ifeq ($(chip), F4)
+    MCU_TARGET = STM32F413
+else ifeq ($(chip), L4)
+    MCU_TARGET = STM32L431
+else
+    $(error ${RED}Invalid MCU specified ${NC}(L4 and F4 are the only options))
```

Comment:
Just tested this on my end; ANSI color codes do not seem to work inside an error. Let me know if you see different results.